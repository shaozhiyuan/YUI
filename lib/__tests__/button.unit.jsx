import renderer from 'react-test-renderer';
import Button from '../button';
import React from 'react';

describe('button', () => {
    it('描述：他是个div', () => {
        const json = renderer.create(<Button/>).toJSON();
        expect(json).toMatchSnapshot();
    });
});