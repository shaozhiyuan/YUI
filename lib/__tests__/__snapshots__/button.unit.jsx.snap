// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 他是个div 1`] = `
<button>
  按钮
</button>
`;
